src/app/login/login.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { Validators } from '@angular/forms';
+import { FormArray } from '@angular/forms';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent {
+  
+  loginForm = this.fb.group({
+    email: ['',[
+      Validators.required,
+      Validators.pattern('^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$')
```

Comment:
As we discussed before, using Validators.email would be better than pattern as it covers more email specific characteristics.